prow/scripts/cluster-integration/kyma-gke-upgrade.sh

```diff
@@ -344,40 +346,59 @@ function checkTestPodTerminated() {
     return 1
 }
 
-createTestResources() {
-    shout "Create e2e upgrade test resources"
-    date
+function installTestChartOrFail() {
```

Comment:
Would it make sense to move this to the `library.sh"`?

Reply:
As discussed offline, we will keep it like that for now.